fe/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -213,6 +214,10 @@
     @VariableMgr.VarAttr(name = LOAD_MEM_LIMIT)
     private long loadMemLimit = 0L;
 
+    // the flags passed to Backends
```

Comment:
defaultRowsetType is only used to represent the storage format.
NOT the heartbeat flag.
Change the comment.

Reply:
done